lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
+            <div class="copy">
+                <p aria-live="polite"></p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="team-edit-fields">
+    <div class="team-required-fields">
```

Comment:
My suggestion for the copy: "Enter information to describe your team. You cannot change these details after you create the team."